tests/synthetic/test_mixtral.py

```diff
@@ -28,17 +28,13 @@ def test_format_conversation_basic(self):
         ]
 
         expected_output = (
-            "<s> [INST] \n"
-            "Hello [/INST]"
-            "Hi there!</s>[INST] "
-            "How are you? [/INST]"
-            "I'm doing well, thank you!</s>[INST] "
+            "<s> [INST] \nHello [/INST]Hi there!</s>[INST] How are you? [/INST]I'm doing well, thank you!</s>[INST] "
```

Comment:
I think original was more readable. wdyt?

Reply:
Yeah I can see that, although for tests I'm less worried about readability.